tutorials/migrating/upgrading_to_godot_4.rst

```diff
@@ -437,6 +437,7 @@ table to find its new name.
 - Shortcut's ``is_valid()`` is now ``has_valid_event()``.
 - TileMap's ``world_to_map()`` is now ``local_to_map()``.
 - TileMap's ``map_to_world()`` is now ``map_to_local()``.
+- Transform2D's ``xform()`` and ``xform_inv()`` are replaced by ``*`` operator for matrix multiplication.
```

Comment:
I'd add some details on that `xform` is `mat * vec` and `xform_inv` is `vec * mat`

Reply:
edited it